<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateMemberInfo</class>
 <widget class="QDialog" name="UpdateMemberInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="memberIDLabel">
       <property name="text">
        <string>Member ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="memberIDUpdate"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="memberNameLabel">
       <property name="text">
        <string>Member Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="memberNameUpdate"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="memberSurnameLabel">
       <property name="text">
        <string>Member Surname</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="memberSurnameUpdate"/>
     </item>
     <item row="3" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Member Sex</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton_maleUpdate">
         <property name="text">
          <string>Male</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_femaleUpdate">
         <property name="text">
          <string>Female</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="dateOfBirthLabel">
       <property name="text">
        <string>Date of Birth</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QDateEdit" name="dateEditUpdate"/>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="membershipDurationLabel">
       <property name="text">
        <string>Membership Duration (Month)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="spinBoxUpdate"/>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
